docs/roles/integrator/exchange-integration.md

```diff
@@ -773,9 +771,92 @@ and we can find its outcome in `receipts_outcome`:
 
 which indicates that the transaction is successful.
 
-## Block and Finality
+## Fungible tokens
+
+Please see the [spec for the fungible token standard](https://nomicon.io/Standards/FungibleToken/README.html) and an [example implementation](https://github.com/near-examples/FT) for reference details.
+
+One notable aspect of the standard is that method names are prefixed with `ft_`. This will be a helpful convention when querying for transactions related to fungible tokens.
+
+### Get balance
+
+Using the abstraction of the NEAR CLI tool, we can check the balance of a user's account with this command:
+
+`near view ft.demo.testnet ft_balance_of '{"account_id": "mike.testnet"}'`
+
+Returns:
+
+```
+View call: ft.demo.testnet.ft_balance_of({"account_id": "mike.testnet"})
+'1000000'
+```
+
+Using HTTPie:
+
+```bash
+http post https://rpc.testnet.near.org jsonrpc=2.0 id=ftbalance method=query \
+params:='{
+  "request_type": "call_function",
+  "finality": "final",
+  "account_id": "ft.demo.testnet",
+  "method_name": "ft_balance_of",
+  "args_base64": "eyJhY2NvdW50X2lkIjogIm1pa2UudGVzdG5ldCJ9"
+}'
+```
+
+Returns:
+
+```bash
+HTTP/1.1 200 OK
+Alt-Svc: clear
+Via: 1.1 google
+access-control-allow-origin:
+content-length: 176
+content-type: application/json
+date: Thu, 27 May 2021 12:53:38 GMT
+
+{
+    "id": "dontcare",
+    "jsonrpc": "2.0",
+    "result": {
+        "block_hash": "3mvNHpZAsXiJ6SuHU1mbLVB4iXCfh5i5d41pnkaSoaJ5",
+        "block_height": 49282350,
+        "logs": [],
+        "result": [
+            34,
+            49,
+            48,
+            48,
+            48,
+            48,
+            48,
+            48,
+            34
+        ]
+    }
+}
+```
+
+As mentioned earlier, the `result` is an array of bytes. There are various ways to convert bytes into a more human-readable form, such as the [dtool CLI](https://github.com/guoxbin/dtool#installation).
+
+`dtool a2h '[34,49,48,48,48,48,48,48,34]' | dtool h2s`
+
+Returns:
+
+`"1000000"`
+
+Note that the fungible token balance of the account `mike.testnet` is `1000000` wrapped in double-quotes. This is because of an issue with JSON serialization. Amounts given in arguments and results must be serialized as Base-10 strings, e.g. "100". This is done to avoid JSON limitation of max integer value of 2**53, which can certainly happen with fungible tokens.
+
+### Simple transfer
```

Comment:
For this section, @volovyk-s please follow the instructions (I think you actually wrote this part) from the FT example. https://github.com/near-examples/FT
Create a fungible token, then send to another user. For instance, this will add storage for `mike.testnet` to the FT contract deployed to `ft.demo.testnet`:

```bash
near call ft.demo.testnet storage_deposit '{"account_id": "mike.testnet", "registration_only": true}' --accountId ft.demo.testnet --amount .025
near call ft.demo.testnet ft_transfer '{"receiver_id": "mike.testnet", "amount": "1000000"}' --accountId ft.demo.testnet --amount 0.000000000000000000000001
```

Now `mike.testnet` has `1000000` tokens.
After that last NEAR CLI command of `ft_transfer`, open up explorer (maybe this will help understand visually) and also get the info of the transaction using NEAR CLI's `tx-status` command. The exchanges will likely be doing lower-level RPC calls that return values like what we get from the `tx-status` command.

Look through the big list of the transaction status and determine how exchanges can tell that this was successful. (I think it's something like Status » SuccessValue or something)

Next, run the same kind of command, except make it error. Maybe send more tokens than are allowed, put in the wrong parameters, or send to an account that doesn't exist or isn't registered. 
Do the same thing where you'll inspect the transaction and look for ways exchanges can tell that this failed. (I think it's something like Status » Failed)

Reply:
Probably makes sense to have subheaders for Successful/Failed transfers, like how I have that below with "Transfer and call"